engine/src/test/scala/cromwell/SimpleWorkflowActorSpec.scala

```diff
@@ -11,7 +11,7 @@ import cromwell.engine.workflow.WorkflowActor
 import cromwell.engine.workflow.WorkflowActor.{StartNewWorkflow, StartWorkflowCommand}
 import cromwell.services.metadata.MetadataService
 import MetadataService.PutMetadataAction
-import cromwell.jobstore.{JobStoreActor, WriteCountingJobStoreDatabase}
+import cromwell.jobstore.{JobStoreActor, WriteCountingJobStore$}
```

Comment:
I have no idea what happened here.  That's now an unnecessary import in both spots anyway.  (shrug)